composer.json

```diff
@@ -18,7 +18,8 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~1.8",
         "phpunit/phpunit": "~4.5",
-        "satooshi/php-coveralls": "~0.6"
+        "satooshi/php-coveralls": "~0.6",
+        "symfony/event-dispatcher": "~2.8|~3.0"
```

Comment:
Shouldn’t it go into `"require"`? We are using the interface which could change in a major version.